config/v1/types_ingress.go

```diff
@@ -54,6 +54,22 @@ type IngressSpec struct {
 	// configurable routes.
 	// +optional
 	ComponentRoutes []ComponentRouteSpec `json:"componentRoutes,omitempty"`
+
+	// requiredHSTSPolicies specifies HSTS policies that are required to be set on
+	// newly created routes matching criteria that are specified in the policy.
+	// - If there are no RequiredHSTSPolicies, any route annotation is valid.
+	// - If the route specifies a HSTS annotation that matches one of the RequiredHSTSPolicies,
+	// a DomainPattern it specifies, and NamespaceSelector it specifies (if any), the route is admitted.
+	// - If the route specifies a HSTS annotation that matches one of the RequiredHSTSPolicies,
+	// a DomainPattern it specifies, but doesn't match a NamespaceSelector it specifies, the route is
```

Comment:
I would have expected that not matching a namespace selector would result in the admission not applying to that route.

Reply:
In this case, "rejected" means the admission doesn't apply to that route.  So, we are covered here, right?